ipykernel/kernelapp.py

```diff
@@ -401,6 +403,18 @@ def init_extensions(self):
                 self.log.debug('ipywidgets package not installed.  Widgets will not be available.')
         # END HARDCODED WIDGETS HACK
 
+    def init_tornado_logger(self):
+        """ Must set up the tornado logger or else tornado will call
+            basicConfig for the root logger which makes the root logger
+            go to the real sys.stderr instead of the capture streams.
+            This function mimics the setup of logging.basicConfig.
```

Comment:
Can you make this docstring follow the format:

* Single line summary
* Blank line
* Paragraph with more information (optional)